packages/console/src/pages/EnterpriseSso/SsoCreationModal/index.tsx

```diff
@@ -54,19 +55,22 @@ function SsoCreationModal({ isOpen, onClose: rawOnClose }: Props) {
 
   const isLoading = !data && !error;
 
-  const { standardConnectors = [], providerConnectors = [] } = data ?? {};
+  const { standardProviders, enterpriseProviders } = useMemo(
+    () => categorizeSsoConnectorProviders(data),
+    [data]
+  );
 
   const radioGroupSize = useMemo(
-    () => getConnectorRadioGroupSize(standardConnectors.length + providerConnectors.length),
-    [standardConnectors, providerConnectors]
+    () => getConnectorRadioGroupSize(standardProviders.length + enterpriseProviders.length),
+    [standardProviders, enterpriseProviders]
   );
```

Comment:
nit: IMO we should revisit these two `useMemo` usages later. Seems unnecessary to me.